packages/@react-spectrum/table/src/Resizer.tsx

```diff
@@ -5,30 +5,38 @@ import {GridNode} from '@react-types/grid';
 // @ts-ignore
 import intlMessages from '../intl/*.json';
 import {MoveMoveEvent} from '@react-types/shared';
-import React, {RefObject, useRef} from 'react';
+import React, {Key, RefObject, useRef} from 'react';
 import styles from '@adobe/spectrum-css-temp/components/table/vars.css';
-import {TableColumnResizeState} from '@react-stately/table';
+import {TableLayoutState, useTableColumnResize} from '@react-aria/table';
 import {useLocale, useLocalizedStringFormatter} from '@react-aria/i18n';
-import {useTableColumnResize} from '@react-aria/table';
-import {useTableContext} from './TableView';
+import {useTableContext, useVirtualizerContext} from './TableView';
 import {VisuallyHidden} from '@react-aria/visually-hidden';
 
 interface ResizerProps<T> {
+  layout: TableLayoutState,
```

Comment:
Do you need to pass the layout via props or could you grab it from the TableContext instead?

Reply:
Doesn't matter, it's an internal component